packages/botonic-core/src/models/legacy-types.ts

```diff
@@ -252,11 +252,13 @@ export interface HubtypeCaseContactReason {
   project_id: string
 }
 
+export interface SessionBot {
```

Comment:
I'm wondering if this is more `InitialSessionBot` rather than `SessionBot` as later this is filled with more values